package.json

```diff
@@ -20,6 +20,9 @@
     "build": "npm-run-all build:clean build:compile",
     "prepack": "npm-run-all build"
   },
+  "engines": {
```

Comment:
Could you move this up a bit, putting it between the `type` and `main` fields? It seems to make a little more sense there.